RESTserver/src/main/java/com/softserve/if072/restservice/service/CategoryService.java

```diff
@@ -47,16 +43,24 @@ public Category getById(int id) throws DataNotFoundException {
         if (category != null) {
             return category;
         } else {
-            throw new DataNotFoundException(String.format(categoryNotFound, id));
+            throw new DataNotFoundException(String.format("Category with id %d not found", id));
```

Comment:
When we will apply localization it will be hardcoded message. So please move to the property file